test.js

```diff
@@ -16,6 +17,10 @@ const NO_NEWLINES_REGEXP = /^[^\n]*$/;
 const STDERR_STDOUT_REGEXP = /stderr[^]*stdout/;
 const TIMEOUT_REGEXP = /timed out after/;
 
+const waitForProcessLaunch = subprocess => {
+	return promisify(subprocess.once.bind(subprocess))('message');
```

Comment:
```suggestion
    return pEvent(subprocess, 'message')
```

I have checked it and my suggestion of using `promisify` actually was wrong. Luckily tests are already using `p-event`, which is even less verbose.